library/src/main/java/com/pokegoapi/api/inventory/Inventories.java

```diff
@@ -33,13 +33,13 @@
 import com.pokegoapi.exceptions.LoginFailedException;
 import com.pokegoapi.exceptions.RemoteServerException;
 import com.pokegoapi.main.ServerRequest;
+import com.pokegoapi.api.player.Stats;
```

Comment:
It is the import for the new class that is located on com.pokegoapi.api.player the old one is located in the same package so before the import was unneeded but now I have to declare the new class location.